docs/modules/ROOT/pages/getting-started/installation.adoc

```diff
@@ -45,10 +45,12 @@ kubectl apply -n kubearchive -f {download-path}/kubearchive.yaml
 ----
 cat <<EOF > /tmp/patch.yaml
 stringData:
-  POSTGRES_URL: <your-postgresql-url>
-  POSTGRES_USER: <kubearchive-username>
-  POSTGRES_DB: <kubearchive-database>
-  POSTGRES_PASSWORD: '<kubearchive-password>'
+  DATABASE_KIND: postgresql
+  DATABASE_PORT:<postgresql-port>
+  DATABASE_URL: <postgresql-url>
+  DATABASE_USER: <kubearchive-username>
+  DATABASE_DB: <kubearchive-database>
+  DATABASE_PASSWORD: '<kubearchive-password>'
```

Comment:
Remove the single quotes for consistency
```suggestion
  DATABASE_PASSWORD: <kubearchive-password>
```

Reply:
I think the password should actually be in quotes as it it could contain characters that would result in invalid YAML, such as a colon.

Reply:
oh, you are right! I'm solving this then